po/pl.po

```diff
@@ -98,7 +98,9 @@ msgid ""
 "write a log with a certain log level (valid values: 1 to 6, for user error, "
 "critical, error, warning, info, and debug respectively)"
 msgstr ""
-"utwórz log używając wybranego poziomu logowania (poprawne wartości: 1 do 6)"
+"utwórz log używając wybranego poziomu logowania (poprawne wartości: 1 do 6, "
+"dla błędów użytkownika, krytycznych, błędów, ostrzeżeń, informacji i "
```

Comment:
Honestly I'm not really happy with this translations. It sounds weird to translate log levels and I have no sensible translation for `debug` :(

Reply:
I hear you! Had the same trouble with the German translations and just went with the English name in the end. Most if not all users will know what that means.

Line 96 with the "fuzzy" should be removed, I reckon.

Reply:
For now I have left `fuzzy` on purpose to give myself some time to think about it. Maybe I will stick to leaving log levels untranslated too.